hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/utils/ResourceLimitCache.java

```diff
@@ -49,20 +49,19 @@ public V get(K key) {
   }
 
   @Override
-  public V put(K key, V value) throws InterruptedException {
+  public void put(K key, V value) throws InterruptedException {
```

Comment:
Remove `ResourceLimitCache.java`.  It is not used anywhere.

Reply:
done